api/v2.0/swagger.yaml

```diff
@@ -18,21 +18,52 @@ securityDefinitions:
 security:
   - basicAuth: []
 paths:
-  /projects/{project_id}/repositories/{repository_id}/artifacts/{artifact_id}:
+  /projects/{project_name}/repositories/{repository_name}/artifacts/{digest}:
     get:
-      summary: Read artifact by id
-      description: endpoint returns artifact by id
+      summary: Get the specific artifact
+      description: Get the artifact specified by the digest under the project and repository
       tags:
         - artifact
-      operationId: readArtifact
+      operationId: get
```

Comment:
The operationId **MUST** be unique among all operations described in the API, operationIds of `get`, `delete` and `list` in `artifact` group are not good practical here, it means other API groups can not use operationId like `get`, `delete` and `list`.

Let's keep the operationIds like the before.

See more https://github.com/go-swagger/go-swagger/commit/8910d8f1bccae424ccbd8b4bb0f4749578a37284 and https://github.com/go-swagger/go-swagger/issues/104

Reply:
Updated